code-challenges.js

```diff
@@ -31,38 +58,101 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+let  givenNum = (temperature1) => {
+  if(temperature1 <= 212) {
+    return `"${temperature1} is below boiling point"`
+  } 
+    else if(temperature2 >= 212) {
+    return `"${temperature2} is above boiling point"`
+  } 
+    else if(temperature3 === 212) {
+    return `"${temperature3} is at boiling point"`
+  } 
+    else {
+    return "something is wrong"
+  }
+}
+console.log(givenNum(temperature1))
+console.log(givenNum(temperature2))
+console.log(givenNum(temperature3))
+//I AM STUCK HERE-----I can't get temp3 to log at boiling point
```

Comment:
Great start to this problem! Couple of things for you to consider, the first being you want to use const to declare a function. Your parameter is like a placeholder and you don't have to use a variable you intend to test. So if we used temp, we could use temp in our conditional statements, as well as in the string interpolation. Lastly, our conditional statement will stop as soon as it is true. By using <= in our first condition to be checked, we would return that first statement for anything less than or equal to 212 which is why you are getting that undesired output for temperature3.

Reply:
So it would be this?:

Pseudo Code:
//Create a function named "givenNumber" that takes in a number named "temp" to see if the given number is below boiling point, at boiling point, or above boiling point
//Boiling point is given number of "212" degrees Fahrenheit
//If the temp is less than boiling point, will return "below boiling point"
//If the temp is above boiling point, will return "above boiling point"
//If the temp is at boiling point, will return at boiling point

const givenNum = (temp) => {
  if(temp < 212) {
    return `"${temp} is below boiling point"`
  } 
    else if(temp > 212) {
    return `"${temp} is above boiling point"`
  }
    else if(temp = 212) {
    return `"${temp} is at boiling point"`
  }  
    else {
    return "something is wrong"
  }
}
console.log(givenNum(temperature1))
console.log(givenNum(temperature2))
console.log(givenNum(temperature3))

Reply:
Nice! You just need to make sure you use === in your 3rd condition being checked but other than that, looks good!